cmd/rekor-cli/app/search.go

```diff
@@ -108,15 +109,7 @@ var searchCmd = &cobra.Command{
 		artifactStr := viper.GetString("artifact")
 		sha := viper.GetString("sha")
 		if sha != "" {
-			var prefix string
-			if !strings.HasPrefix(sha, "sha256:") && !strings.HasPrefix(sha, "sha1:") {
-				if len(sha) == 40 {
-					prefix = "sha1:"
-				} else {
-					prefix = "sha256:"
-				}
-			}
-			params.Query.Hash = fmt.Sprintf("%v%v", prefix, sha)
+			params.Query.Hash = util.PrefixSHA(sha)
```

Comment:
Is this still needed here? Probably if folks update the cli but not the server?

Reply:
Right, safe to keep!